Test/LocalizationTest.cs

```diff
@@ -62,8 +62,8 @@ public static void TestLocalization()
             f.Run();
 
             // Was single peak!!!
-            Assert.AreEqual(0, newPsm.MatchedIonDictOnlyMatches[ProductType.BnoB1ions].Count(b => b > 0));
-            Assert.AreEqual(1, newPsm.MatchedIonDictOnlyMatches[ProductType.Y].Count(b => b > 0));
+            Assert.AreEqual(0, newPsm.MatchedIonDictOnlyMatchesMasses[ProductType.BnoB1ions].Count(b => b > 0));
+            Assert.AreEqual(1, newPsm.MatchedIonDictOnlyMatchesMasses[ProductType.Y].Count(b => b > 0));
             // If localizing, three match!!!
```

Comment:
Can you add a check here that the matchedIonIntensities are 0 and 1, respectively?